packages/paste-core/components/button/src/styles.ts

```diff
@@ -93,26 +93,66 @@ export const SizeStyles: {[key in ButtonSizes]: BoxStyleProps} = {
 
 export const ToggleStyles: BoxStyleProps = {
   transition: 'background-color 150ms ease-in, box-shadow 150ms ease-in, color 150ms ease-in',
+  color: 'colorText',
+  backgroundColor: 'colorBackgroundBody',
+  boxShadow: 'shadowBorderWeak',
```

Comment:
issue: Pretty sure this shouldn't be applied here, because it introduces a border on icon only toggle buttons, where it shouldn't

before: 
<img width="250" alt="image" src="https://user-images.githubusercontent.com/368249/222010977-f9a3ba7a-ff4a-414e-9bac-c18bf9076c0f.png">

after:
<img width="246" alt="image" src="https://user-images.githubusercontent.com/368249/222010956-818b679b-cfa8-45cc-87e9-c7fdd63a9e1c.png">